.gitignore

```diff
@@ -3,4 +3,36 @@
 *.suo
 /Debug
 /GeneratedFiles
-build
\ No newline at end of file
+build

```

Comment:
On 09/04/15 22:32, Matt Broadstone wrote:
> In .gitignore
> <https://github.com/mbroadst/qamqp/pull/26#discussion_r28057154>:
> 
>> \ No newline at end of file
>> +build
> 
> lol I think you'll find your "git life" much easier if you opt for
> shadow builds, e.g.:
> 
> |$ cd qamqp
> $ mkdir build && cd build
> $ qmake ..
> $ make -j5
> |
> 
> All tests are rpathed to the libraries built, so you can then just do a
> |make check| to run all tests

Ahh okay, didn't know qmake-generated Makefiles did that.  I've been
using CMake for this project here, which does out-of-tree builds by
default, but yes, I'm used to traditional Makefile projects.

Again, my C heritage showing. :-)

-- 
Stuart Longland (aka Redhatter, VK4MSL)

I haven't lost my mind...
  ...it's backed up on a tape somewhere.